sqladmin/application.py

```diff
@@ -594,22 +594,21 @@ async def _handle_form_data(self, request: Request, obj: Any = None) -> FormData
         """
 
         form = await request.form()
-        form_data: Dict[str, Any] = {}
-
-        for key, value in form.items():
+        form_data: List[Tuple[str, Union[str, UploadFile]]] = []
+        for key, value in form.multi_items():
             if not isinstance(value, UploadFile):
-                form_data[key] = value
+                form_data.append((key, value))
                 continue
 
             should_clear = form.get(key + "_checkbox")
             empty_upload = len(await value.read(1)) != 1
             if should_clear:
-                form_data[key] = None
+                continue
```

Comment:
I haven't checked this in details, shouldn't this be `form_data[key] = []` instead of `continue`?

Reply:
Changed type of `form_data` from `Dict` to `List[Tuple[str, Union[str, UploadFile]]]` 
so assuming you meant to change it to `form_data.append((key,[]))`
Ran a quick local test and none of the unitests have failed, so either there isn't a difference or tests aren't covering everything.
Can push the difference if you want.

Reply:
yeah I think coverage is missing, I will add it in a separate MR.

Reply:
Can we change it please?

Reply:
Just ran lint and I think its supposed to be `continue`. In starlette, the class is 
```
class FormData(ImmutableMultiDict[str, typing.Union[UploadFile, str]]):
    """
    An immutable multidict, containing both file uploads and text input.
    """

    def __init__(
        self,
        *args: typing.Union[
            "FormData",
            typing.Mapping[str, typing.Union[str, UploadFile]],
            typing.List[typing.Tuple[str, typing.Union[str, UploadFile]]],
        ],
        **kwargs: typing.Union[str, UploadFile],
    ) -> None:
        super().__init__(*args, **kwargs)
```

lint check fails with

1. `Argument 1 to "append" of "list" has incompatible type "Tuple[str, List[<nothing>]]"; expected "Tuple[str, Union[str, UploadFile]]` if I pass `[]`
2. `Argument 1 to "append" of "list" has incompatible type "Tuple[str, None]"; expected "Tuple[str, Union[str, UploadFile]]"` if I pass `None`